packages/react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,80 +1,288 @@
+        <a aria-label={label} className={linkClass} href={url}></a>
+      )}
+    </li>
+  );
+};
+
+const Breadcrumb: FC<BreadcrumbProps> = ({ className, links, buttonLabel }) => {
+  const { prefix } = useGlobalSettings();
+
+  // State
+  const [isCtxMenuOpen, setIsCtxMenuOpen] = useState(false);
+  const [ctxMenuPosition, setCtxMenuPosition] = useState({
+    start: 0,
+    top: 0,
+  });
+  const [isMenuCollapsed, setIsMenuCollapsed] = useState(false);
+
+  // Refs
+  const breadcrumbsRef = useRef<HTMLOListElement>(null);
+  const breadcrumbsWidth = useRef<number>(0);
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const ctxMenuRef = useRef<HTMLOListElement>(null);
+
+  // Shortcuts to classnames
+  const baseClass = `${prefix}--breadcrumb`;
+  const innerClass = `${baseClass}--inner`;
+  const itemsClass = `${baseClass}--items`;
+  const itemClass = `${baseClass}--item`;
+  const itemFirstClass = `${itemClass}__first`;
+  const contextClass = `${baseClass}--context`;
+  const contextCollapsedClass = `${contextClass}__collapse`;
+  const contextId = `${baseClass}--menu`;
+  const buttonClass = `${contextClass}--button`;
+
+  // Dynamic classnames
+  const breadcrumbClass = classnames(baseClass, className);
+  const contextItemsClass = classnames(contextClass, {
+    [contextCollapsedClass]: isMenuCollapsed,
+  });
+  const contextMenuClass = classnames(`${contextClass}--menu`, {
+    [`${contextClass}--menu__visible`]: isCtxMenuOpen,
+  });
+
+  // Handlers
+  function handleButtonClick() {
+    setIsCtxMenuOpen(!isCtxMenuOpen);
+  }
+
+  const collapseMenu = useCallback(
+    (collapse: boolean) => {
+      if (collapse && !isMenuCollapsed) {
+        setIsMenuCollapsed(true);
+      }
+
+      if (!collapse && isMenuCollapsed) {
+        setIsMenuCollapsed(false);
+      }
+    },
+    [isMenuCollapsed]
+  );
+
+  // Get the width of the breadcrumbs and store it in a ref
+  useLayoutEffect(() => {
+    const breadcrumbs = breadcrumbsRef?.current;
+    if (!breadcrumbs || !window || !document) {
+      return;
+    }
+    breadcrumbsWidth.current = breadcrumbs.offsetWidth;
+  }, []);
+
+  // Collapse or uncollapse the Context Menu depending on body width
+  useLayoutEffect(() => {
+    const breadcrumbs = breadcrumbsRef?.current;
+
+    if (!breadcrumbs || !window || !document) {
+      return;
+    }
 
-  const handleScreenResize = () => {
-    enableContextMenu(window.innerWidth <= 1024); // turn on context menu when screen is resized to a smaller size
-  };
+    const observer = new ResizeObserver((entries) => {
+      const bodyWidth = entries[0].contentRect.width;
 
+      if (breadcrumbsWidth.current >= bodyWidth / 1.5) {
+        collapseMenu(true);
+      } else {
+        collapseMenu(false);
+      }
+    });
+
+    observer.observe(document.body);
+
+    return () => {
+      observer.disconnect();
+    };
+  }, [collapseMenu]);
+
+  // Position the Context Menu right below the button
+  useLayoutEffect(() => {
```

Comment:
I have this issue with the breadcrumbs context placment

https://github.com/international-labour-organization/designsystem/assets/36326203/d39e51c0-6d22-4025-8b09-928826da1622

Testewed with:
- Chrome v124
- Firefox v125
- Safari

Reply:
Hey @GGKapanadze, it turns out this was being caused by the Canvas wrapper that appears in the Docs page. To fix it, I'm just rendering the component in Storybook outside of a Canvas box and then showing the code beneath it using one of the native Storybook blocks.

The issue is that the Canvas container has a `transform` property which creates its own [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). This in turn breaks the positioning of the Context Menu, which is expecting to calculate its position from the viewport and instead ends up calculating from the Canvas box.

Because I think it's very unlikely that a Breadcrumb would ever need to be rendered in a container with a transform set on it, I think we can let this go. But I did point out in the docs that that could have some undesirable consequences.